api/report_server.thrift

```diff
@@ -21,58 +21,55 @@ const string API_VERSION = '6.0'
 const i64 MAX_QUERY_SIZE = 500
 //=================================================
 
-//-----------------------------------------------------------------------------
-struct RunData{
+struct RunData {
   1: i64            runId,        // unique id of the run
   2: string         runDate,      // date of the run
   3: string         name,         // human readable name of the run
   4: i64            duration,     // duration of the run; -1 if not finished
   5: i64            resultCount,  // number of results in the run
   6: string         runCmd,       // the used check command
-  7: optional bool  can_delete    // true if codeCheckerDBAccess::removeRunResults()
+  7: optional bool  canDelete,    // true if codeCheckerDBAccess::removeRunResults()
```

Comment:
Do we still need the `canDelete`? I'm not sure we use it in the database with the new schema.